README.md

```diff
@@ -95,13 +118,9 @@ As you can see, you can mix and match dsc resources with common puppet resources
 All [puppet metaparameters](https://docs.puppetlabs.com/references/latest/metaparameter.html) should also be supported.
 
 ## Limitations
-- DSC Composite resources not yet fully supported.
+- DSC Composite Resources are not yet fully supported.
 - PS Objects like 'PSCredential' as parameters value not yet supported.
 
 ## Notes
-The puppet types are build from the source code of the DSC Resources MOF schema files.
-If you want the build Puppet types for your own custom DSC Resources, read the [README_BUILD](https://github.com/msutter/puppet-dsc/blob/master/README_BUILD.md)
-
-## License
```

Comment:
I'm less concerned about removing the license mention, since the repo has a
matching license file.

I'm more concerned about the "copyright Marc Sutter" bit.  I just want to
ensure we don't just rip off his name without appropriate attribution, etc.

-e
On Aug 11, 2015 11:15 AM, "James Pogran" <notifications@github.com> wrote:

> In README.md
> <https://github.com/puppetlabs/puppetlabs-dsc/pull/55#discussion_r36778591>
> :
>
> >  - PS Objects like 'PSCredential' as parameters value not yet supported.
> >
> >  ## Notes
> > -The puppet types are build from the source code of the DSC Resources MOF schema files.
> > -If you want the build Puppet types for your own custom DSC Resources, read the [README_BUILD](https://github.com/msutter/puppet-dsc/blob/master/README_BUILD.md)
> > -
> > -## License
>
> Yes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/puppetlabs-dsc/pull/55/files#r36778591>.
>